rct_optimizations/include/rct_optimizations/types.h

```diff
@@ -47,20 +47,30 @@ struct Pose6d
 // Useful typedefs shared by calibrations
 struct Correspondence2D3D
 {
+  Correspondence2D3D()
```

Comment:
Would it also be useful to have a constructor with fields that initialize `in_target` and `in_image`? For example:
```
Correspondence2D3D(const Eigen::Vector3d& pt_target, const Eigen::Vector2d& pt_image) {...}
```